### ISSUE TITLE
When first added opens screen asking for data with no field discriptions

### ISSUE THREAD

**Issue by Dazula** [markgdev/home-assistant_OctopusAgile#38]
![Picture 1](https://user-images.githubusercontent.com/76919589/103694169-6304a380-4f92-11eb-95d8-d344383cc0ba.png)

Home Assistant 2020.12.2

**Comment by markgdev** [comment]
Is this when you install through HACS? Where do you see this screen?
If it’s in the add integrations page, installation is performed through the configuration.yaml file.

**Comment by Dazula** [comment]
> Is this when you install through HACS? Where do you see this screen?
> 
> If it’s in the add integrations page, installation is performed through the configuration.yaml file.

Apologises I should have provided more information.

It was after a manual install.  I copied files to components. The window popped up when adding the integration within HA.

I simply entered junk text in the pop up and it worked fine.

Edited configuration.yaml with my api etc.

**Comment by markgdev** [comment]
No problem, I think I can see why this is happening, will have a look to try and replicate and stop it.

**Comment by markgdev** [comment]
Just having a look, did you go to "Add integration" when you got this issue? Or did it pop up before that point?

**Comment by Dazula** [comment]
Yes correct - At point of Add integration.  (I also tried 'Add integration' this after installing via HACS and same happened). I guess I was more puzzled by the empty field names  - Have in mind I set up configuration.yaml before installing my HACS

**Comment by markgdev** [comment]
That makes sense. It's because I hadn't setup the ability to add the integration through the GUI. There isn't any need to go to Add Integrations as you have already configured the integration through configuration.yml.

**Comment by HACS-bank** [comment]
I was also taken in by this - it seemed "obvious" to me that the dialogue was asking for MPAN, MeterID and API_key ...
